include/oneapi/tbb/detail/_config.h

```diff
@@ -165,6 +165,10 @@
         #define __TBB_ipf 1
     #elif __i386__||__i386  // __i386 is for Sun OS
         #define __TBB_x86_32 1
+    #elif __powerpc64__ || __ppc64__
```

Comment:
Can you make sure it still works for `__powerpc64__` we don't test that configuration regularly.

Reply:
@reble If you mean whether macro we be applied to `__powerpc64__` case, then yes, sure. I cannot test `__powerpc64__` system, however, since I have none. Darwin ppc64 is defined as `__ppc64__`.

P. S. Sorry for a long delay, I am just back to PPC hardware after 4 months.

Reply:
@reble Looks like it is not needed, I just dropped it. Generic arch is defined, but not used anywhere in the code, so let it be generic.

I have also dropped all header changes, you were absolutely right, those are redundant. Thank you for pointing that out.